Source/Core/Core/ConfigManager.cpp

```diff
@@ -502,8 +485,6 @@ void SConfig::LoadDisplaySettings(IniFile& ini)
   display->Get("RenderWindowHeight", &iRenderWindowHeight, 480);
   display->Get("RenderWindowAutoSize", &bRenderWindowAutoSize, false);
   display->Get("KeepWindowOnTop", &bKeepWindowOnTop, false);
-  display->Get("ProgressiveScan", &bProgressive, false);
```

Comment:
IIRC, this is also used for the GameCube (which is why it is in Display and not in the Sysconf section).
Not sure how to deal with the fact that the SYSCONF overrides this setting though... maybe we should make this one override the SYSCONF setting instead? (Or is it worth exposing this setting in an INI at all?)